test cases/common/198 dict/meson.build

```diff
@@ -0,0 +1,12 @@
+project('dict test', 'c')
+
+dict = {foo : 'bar',
```

Comment:
How do I specify key with spaces?

Reply:
you don't for now, it was a bit late and I just reused the kwargs code path, I think I'll change that :)

Reply:
Fixed

Reply:
@nirbheek , you don't for now, I don't know if we'll want that later, I've just made it so you can only use literal strings as keys

Reply:
Probably better to match Python's style and have the syntax be `{'foo': 'bar'}` then? So we can extend it later.

Reply:
yes that's what I did, check the updated tests ;)